libbeat/dashboards/import_dashboards.go

```diff
@@ -0,0 +1,559 @@
+package main
```

Comment:
I think for the moment it is great to have it in one file to easily make changes and push the project forward. If we move forward with the project and start to ship it with beats. we should start to think about where to place these files as it feels kind of strange to have a golang project with a main method inside libbeat.